projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -384,7 +384,7 @@ public function add_jetpack_menu() {
 		$this->add_admin_menu_separator( $position++, 'manage_options' );
 
 		// TODO: Replace with proper SVG data url.
-		$icon = "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='20' viewBox='0 0 32 32' %3E%3Cpath fill='%23a0a5aa' d='M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16s16-7.2,16-16S24.8,0,16,0z'%3E%3C/path%3E%3Cpolygon fill='%23fff' points='15,19 7,19 15,3 '%3E%3C/polygon%3E%3Cpolygon fill='%23fff' points='17,29 17,13 25,13 '%3E%3C/polygon%3E%3C/svg%3E";
+		$icon = "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='20' viewBox='0 0 40 40' %3E%3Cpath fill='%23a0a5aa' d='M20 0c11.046 0 20 8.954 20 20s-8.954 20-20 20S0 31.046 0 20 8.954 0 20 0zm11 17H21v19l10-19zM19 4L9 23h10V4z'/%3E%3C/svg%3E";
```

Comment:
Instead of adding the new icon here, could we change the jetpack font-icon implementation
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/scss/jetpack-icons.scss#L23-L26

with the SVG one?

Reply:
Possibly, but we want the image set as the icon of the menu item so that it is returned in the API call that is used by Calypso. I think the principle being that if you want a custom icon, that isn't part of dashicons, then you need to use an SVG.

Reply:
> I think the principle being that if you want a custom icon, that isn't part of dashicons, then you need to use an SVG.

That's exactly what I mean, let's use an svg icon for Jetpack and discard any CSS.

Reply:
Oh I see, so this is the default approach for Jetpack? Yes, we could do that. I need to get Jetpack Design to take a look at this anyway.

Reply:
Looking at this a bit more, I think the use of the font means you can inherit the colours, for things like hover states, without having to use CSS masks etc. 

We'd need to make the other changes that have been added to the admin menu for the Jetpack logo, and that feels like too big a change to me, especially as the approach to get this working for the admin menu feels sub-optimal. 

It actually seems like something that should be changed in core. Not being able to have the icons pick up on the colour of the menu item is a drawback to using SVG/data URIs. I'll sort out opening an issue to track it.

Reply:
I agree, let's proceed with merging this and we can iterate in the future. Thanks for researching it!

Reply:
thanks, Harris. 🙇‍♂️